addons/viewport/src/components/tests/RotateViewport.spec.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { shallow } from 'enzyme'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Please use the .test.js naming convention here too

Reply:
No use of .jsx file extention

Reply:
Should I change all `.jsx` files to be `.js`?

Reply:
Yes please!

Reply:
If you can rename all `.spec.` to `.test.` and all `.jsx` to `.js` I think this is good to go!

Let me know if you need any help! 👍